cmd/sidecar_mounter/main.go

```diff
@@ -189,5 +192,12 @@ func prepareMountConfig(sp string) (*sidecarmounter.MountConfig, error) {
 		return nil, fmt.Errorf("failed to fetch bucket name from CSI driver")
 	}
 
+	for _, opt := range mc.Options {
+		if strings.Contains(opt, "experimental-local-file-cache") {
```

Comment:
If we only change the temp path when the `experimental-local-file-cache` is passed, it will not help solving https://github.com/GoogleCloudPlatform/gcs-fuse-csi-driver/issues/57, because for manual sidecar injection, the second cache volume may exist, we also want to change the temp dir value even without the `experimental-local-file-cache` option.

Reply:
Resolving as discussed offline. This feature no longer necessitates driver changes.